### ISSUE TITLE
Separate background color per terminal in split kitty

### ISSUE THREAD

**Issue by dagbdagb** [kovidgoyal/kitty#6971]
**Is your feature request related to a problem? Please describe.**
I typically run 9 workspaces with fullscreen split kittys on each workspace.  Ideally, I use each workspace for 1 particular thing am I working on, at any given time, logging in to a handful of hosts via ssh at the same time. What hosts and how many shells varies. My issue is keeping track of where I am logged in, in each terminal on each workspace. So, not a problem with kitty, just one of getting more visual cues from my work environment.

**Describe the solution you'd like**
Being able to quickly identify what host I am logged in to on any given terminal by adopting some color scheme for the individual terminal background color would be very useful. Also very useful in the zoomed out 'show all workspaces' view.

Hence, a two-step solution may appear practical:
1. the ability to adopt a separate background color (or theme) per terminal in a split window.
2. the ability for kitty to pick up something which is executed or set by .bashrc on the remote host, to set the background color.
Can one use custom escape codes in PS1 for this? No idea, but that could enable this feature also on shells/platforms where we cannot trivially install/execute custom scripts or binaries.

**Describe alternatives you've considered**
Setting the theme for the entire window/session/workspace does not handle the "per-remote-host" angle I am looking for.
I tried looking at solving background colors with escape codes on individual hosts, but have not figured a practical way to do this yet.

**Additional context**
In short, this may look like a way to a)apply themes per split and b)triggering it via some 'in-band' mechanism.

And I am not entirely sure if what I am asking for is even reasonable to expect from kitty, or if the correct solution is to fix it with 
escape-codes on each remote host. I am confident you have an opinion on this.

Thank you for kitty.

**Comment by kovidgoyal** [comment]
There are standard escape codes you can use to change terminal
background colors, use them. Or use the kitty remote control feature.
